app/src/components/admin-page/edit-product-modal.vue

```diff
@@ -69,22 +69,14 @@
                                 <v-col>
                                     <v-autocomplete
                                         v-model="editProduct.manufacturerID"
-                                        :items="manufacturers"
+                                        :items="manufacturerSearchItems"
```

Comment:
Everything works just perfekt for tags!
Well done :tada: 

But somethings seems to be wrong for manufacturers as I can't create a single one :shrug:

Reply:
Should be fixed by new commit